### DISCUSSION TITLE
How to connect to Tensorboard?

### DISCUSSION THREAD

**Discussion by GsHeri** [gitmylo/audio-webui#150] [Q&A]
Hey there,
how can i hook up tensorboard to the rvc model training process? 
i tried pointing it to the /data/training/rvc/modelname folder, but it doesn't seem to recognize any checkpoints or progress.
do i have to use special options? another folder? is it maybe not supported at all?
i'll be happy to hear your answers :)
Thanks

**Comment by gitmylo** [comment]
Tensorboard is not explicitly implemented, and likely not supported. References to tensorboard are from the original RVC source code, as i kept a lot of code intact.

**Reply by GsHeri** [reply]
okayyy so what does that mean exactly?
do you think it might be possible to attach to the underlying rvc process somehow? (if so... would you happen to have an idea where to start looking?)
basically, rvc does create events.out.tfevents out of the box (i do have some from back when i trained via mangio-rvc), so i thought this implementation surely has to export those as well. but it seems it doesn't. how come? did you... uhmm.. turn them off? :D do you have to explicitly activate the creation of them? is there possibly an option somewhere to generate them?
just trying to understand if there's a chance to get it running, since it's a neat tool to have, and it seems to be working with vanilla, so i was wondering why it doesn't with this.
thanks for your help <3

**Reply by gitmylo** [reply]
I'm not sure, as i haven't really used tensorboard myself before.
